WooCommerce/Classes/POS/Utils/StockFormatter.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+import struct Yosemite.POSSimpleProduct
+
+struct StockFormatter {
+    static func stockStatusLabel(for product: POSSimpleProduct) -> String {
+        switch product.manageStock {
+        case false:
+            return product.productStockStatus.description
+        case true:
+            guard let stock = product.stockQuantity else {
+                return product.productStockStatus.description
+            }
+            if stock <= 0 {
+                return Localization.outOfStock
+            } else {
+                return String.localizedStringWithFormat(Localization.inStockWithQuantity, "\(stock)")
+            }
+        }
+    }
+
+    enum Localization {
```

Comment:
nit: this could be private?
```suggestion
    private enum Localization {
```

Reply:
Yup! https://github.com/woocommerce/woocommerce-ios/pull/15670/commits/c9b8497a23dace151763bb652fc43cd7872ae888